index.html

```diff
@@ -10,6 +10,10 @@
       name="description"
       content="Totality is a simple CSS frameowrk to kick start your frontend project"
     />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+      rel="stylesheet"
+    />
```

Comment:
you can import this in your main CSS file. so that you don't need to import it in each page in html

Reply:
yes okay